src/scripts/main.js

```diff
@@ -815,6 +819,15 @@
                     quantity: result.quantity,
                 });
             }
+            else if (result.type in our_map) {
+                items.push({
+                    id: our_map[result.type],
+                    type: result.type,
+                    name: result.name,
+                    pluralized_name: result.pluralized_name || '',
```

Comment:
Should probably do this for the other pluralized name

Reply:
added